containers/daap-python-base/src/var/task/glue_and_athena_utils.py

```diff
@@ -9,67 +9,160 @@
 athena_client = boto3.client("athena")
 
 
-def create_glue_database(
-    glue_client: BaseClient, database_name: str, logger: DataPlatformLogger
-):
+def create_database(
```

Comment:
I think this change might have broken some tests on main, as it no longer gets passed the glue client